content/v2.0/query-data/guides/monitor-states.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Monitor states
+seotitle: Monitor states and state changes in your events and metrics with Flux.
+description: Flux provides several functions to help monitor states and state changes in your data.
+v2.0/tags: [states, monitor, flux]
+menu:
+  v2_0:
+    name: Monitor states
+    parent: How-to guides
+weight: 209
+---
+
+Flux helps you monitor states in your metrics and events:
+
+- [Find how long a state persists](#find-how-long-a-state-persists)
+- [Count the number of states](#count-the-number-of-states)
```

Comment:
Maybe `Count the number of consecutive states`?

Reply:
at first, I had "Count the number of states in the interval," but thought it may be too long...technically, it keeps counting states throughout the interval (after a 'reset' or new state)